certified-connectors/Yarado/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "{{ clientId }}",
+          "scopes": [
+            "robots task-files task-schedules"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://api.yarado.com/oauth/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://api.yarado.com/oauth/token"
+            },
+            "refreshUrl": {
+              "value": "https://api.yarado.com/oauth/refresh"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#f8aaaa",
+    "capabilities": [],
+    "publisher": "Yarado"
```

Comment:
@MarcinKowalczukYarado stackOwner property is missing, please refer to [this template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json).